__tests__/app.test.js

```diff
@@ -65,10 +65,86 @@ describe("GET /api/articles/:article_id", () => {
   test("requests for invalid article IDs are rejected", () => {
     return request(app)
       .get("/api/articles/999999")
-      .expect(404)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe("999999 is not a valid article ID.");
+      });
+  });
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("article votes can be increased", () => {
+    const addedVotes = { inc_votes: 55 };
+    return request(app)
+      .patch("/api/articles/9")
+      .send(addedVotes)
+      .expect(200)
+      .then(({ body }) => {
+        expect(Object.keys(body).length).toBe(1);
+        expect.objectContaining({
+          title: "They're not exactly dogs, are they?",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "Well? Think about it.",
+          created_at: 1591438200000,
+          votes: 55,
+        });
+      });
+  });
+  test("article votes can be decreased", () => {
+    const addedVotes = { inc_votes: -55 };
+    return request(app)
+      .patch("/api/articles/9")
+      .send(addedVotes)
+      .expect(200)
+      .then(({ body }) => {
+        expect(Object.keys(body).length).toBe(1);
+        expect.objectContaining({
+          title: "They're not exactly dogs, are they?",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "Well? Think about it.",
+          created_at: 1591438200000,
+          votes: -55,
+        });
+      });
+  });
+  test("article votes unchanged if inc_votes is 0", () => {
```

Comment:
i'm a bit confused about this test

it's targeting the same article with the same `inc_votes` but it's expecting a different result!

You are getting a false positive from your `objectContaining` mistake (see previous comment)

Needs a refactor